fedimint-core/src/util/mod.rs

```diff
@@ -320,41 +324,41 @@ pub fn handle_version_hash_command(version_hash: &str) {
 pub async fn retry<F, Fut, T>(
```

Comment:
Unclear if we even need it anymore. Seems like any future is retriable now? https://docs.rs/backon/latest/backon/trait.Retryable.html#tymethod.retry

Reply:
we don't get logging, also that uses `tokio::time::sleep` (read hates wasm)

Reply:
also (|| async { }).retry(...) feels very awkward

Reply:
True.